tests/Console/ConfigurationResolverTest.php

```diff
@@ -1143,6 +1143,47 @@ public function provideDeprecatedFixerConfiguredCases()
         ];
     }
 
+    public function provideGetDirectoryCases()
+    {
+        return [
+            [null, '/my/path/my/file', 'path/my/file'],
```

Comment:
Shouldn't the expected relative path be `my/file` instead of `path/my/file`?

Reply:
Maybe in the case where there is no cache file,`getDirectory` doesn't behave as expected. But it's the current behavior. Fixing this potential issue is not the goal of this PR and I'm not sur this wouldn't be considered as a BC break.